<?xml version="1.0"?>

<launch>
<rosparam command="load" file="$(find graph_plot)/config/test.yaml" ns="plot_paths"/>
<node pkg="graph_plot"  type="plot_trajectory.py" name="plot_paths" output="screen">
</node>
</launch>
